src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int index = random.nextInt(Colors.values().length);
+        Colors colors = Colors.values()[index];
```

Comment:
This is what happens when previous comment is not followed. You have one color in variable but type and variable name are in plural what causes confusion
```suggestion
        Color color = Color.values()[index];
```

Reply:
done